### ISSUE TITLE
Adding new language lacking/outdated documentation

### ISSUE THREAD

**Issue by schafferer** [explosion/spaCy#1744]
I'm following the guide on how to add a new language, and the documentation seems to be outdated/lacking. 
The 'Training a new language model'-section says to use word_freqs.py from the dev resources, but nothing about the input format or usage. The script seems to require some bz2 files, but the documentation states that it requires unprocessed text files. 

What do i really need to do here?

Please see [issue 23](https://github.com/explosion/spacy-dev-resources/issues/23) from spacy-dev-resources.

**Comment by machakux** [comment]
I think both the documentation for [adding languages](http://spacy.io/usage/adding-languages) (mostly the training section) and training scipts  mentioned in that documentation are probably a bit outdated or lacking.

For example;
In generating word frequencies section, the current `word_freqs.py` script included seem to produce merged file with two columns instead of three as required according to docs.

The training section doesn't give much insights on how to use word frequencies and vector files generated in training language models. Probably the documentation could mention usage of utilities like `spacy init-model` command

Also in training vectors the `word_vectors.py` script  seem to produce **binary** vector file but `spacy init-model`  command seem to accept only Gensim vectors in **text** format. This is ok but I think it could be better if the sample script could produce a vector file which is compatible with `spacy init-model`  command.

I also think dependancies like Gensim which need to be installed first in order to execute some scripts are not explicitly mentioned in the docs.

**Comment by machakux** [comment]
I modified the `word_freqs.py` to fix some errors that I was getting, accept a directory containing input text files as `input_loc` and give output file with three columns as specified in the docs.

I also modified `word_vectors.py` to output both text and binary vectors.

The modified scripts can be found at https://gist.github.com/machakux/8814f9927498c34a08635820088fd55b

I am not sure if I have done it correctly but it seems to work for me so far except the `spacy init-model` command expects a word frequency file to contain quoted strings but the documentation doesn't say so therefore I faced an issue https://github.com/explosion/spaCy/issues/1209 . It seems the issue can be resolved by modifying the script to add quotes on words or creating a custom model initialization command that doesn't wrap words with `ast.literal_eval`.
